client/src/pages/workspace/mutableInput/MutableInput.component.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useRef, useEffect, ChangeEvent, KeyboardEvent } from "react";
+import OutsideClickHandler from 'react-outside-click-handler';
+
+import "./MutableInput.style.css";
+import {MutableInputProps} from "./props";
+
+const MutableInput: React.FC<MutableInputProps> = ({ text, textSet, placeHolder }) => {
+    const currentField = useRef<HTMLTextAreaElement>(null);
+    const [edit, editSet] = useState<boolean>(false);
```

Comment:
```suggestion
    const [edit, setEdit] = useState<boolean>(false);
```
However in general, it's a bad name because it's a verb but type is boolean

Reply:
You fixed only in this case. Make the same change in other places